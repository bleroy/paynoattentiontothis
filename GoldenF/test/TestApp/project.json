{
  "version": "1.0.0-*",
  "buildOptions": {
    "compilerName": "fsc",
    "compile": {
      "includeFiles": [
        "Test.fs"
      ]
    }
  },
  "dependencies": {
    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160316",
    "Microsoft.NETCore.App": {
      "type": "platform",
      "version": "1.0.0-rc2-3002702"
    },
    "xunit": "2.1.0",
    "dotnet-test-xunit":"1.0.0-rc2-build10015",
    "Library": {
      "target": "project",
      "version": "1.0.0"
    }
  },
  "tools": {
    "dotnet-compile-fsc": {
      "version": "1.0.0-*",
      "imports": [
        "dnxcore50",
        "portable-net45+win81",
        "netstandard1.3"
      ]
    }
  },
  "testRunner": "xunit",
  "frameworks": {
    "netcoreapp1.0": {
      "imports": [
        "portable-net45+win8",
        "dnxcore50",
        "dotnet5.4"
      ]
    }
  }
}
