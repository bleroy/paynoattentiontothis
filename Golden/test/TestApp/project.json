{
  "version": "1.0.0-*",

  "dependencies": {
    "Library": "1.0.0-*",
    "Microsoft.NETCore.App": {
      "type": "platform",
      "version": "1.0.0-rc2-3002702"
    },
    "xunit": "2.1.0",
    "dotnet-test-xunit": "1.0.0-rc2-*"
  },
  "testRunner": "xunit",
  "frameworks": {
    "netcoreapp1.0": {
      "imports": [
        "dotnet5.4",
        "portable-net451+win8"
      ]
    }
  }
}
